src/Debug.c

```diff
@@ -46,7 +47,39 @@ void Ros_Debug_BroadcastMsg(char* fmt, ...)
 
     if (ros_DebugSocket == -1)
         Ros_Debug_Init();
-
+    // Timestamp
+    //The timestamp for the message "Found Micro-Ros PC Agent" will be the epoch time (THU 1970-01-01 00:00:00.000) as the global flags 
+    //are set to indicate that the Micro-Ros PC Agent is connected but the first sync of the host time using the micro-ROS agent is yet to occur
+    struct tm synced_time;
+    struct timeval tv;
+    int64_t nanosecs = rmw_uros_epoch_nanos();
```

Comment:
This is declared a second time below inside the IF statement. Please remove this instance so that there is only call to `rmw_uros_epoch_nanos`. This will also remove the duplicated declaration.

Reply:
I will remove that declaration of rmw_uros_epoch_nanos.